srcpkgs/rust/patches/0010-Fix-dynamic-linkage-of-musl-libc-for-the-libc-crate.patch

```diff
@@ -1,33 +1,31 @@
-From e363a0e0ff7ffc8bd83f72ec1a20d289996b6370 Mon Sep 17 00:00:00 2001
+From 7e100ca9dad23efa1aafccd8faf86ec5fa85c351 Mon Sep 17 00:00:00 2001
 From: q66 <daniel@octaforge.org>
 Date: Sun, 3 May 2020 18:02:03 +0200
-Subject: [PATCH 12/15] Fix dynamic linkage of musl libc for the libc crate
+Subject: [PATCH 10/15] Fix dynamic linkage of musl libc for the libc crate
 
 ---
- vendor/libc/src/lib.rs      | 1 +
  vendor/libc/src/unix/mod.rs | 6 +++---
- 2 files changed, 4 insertions(+), 3 deletions(-)
+ 1 file changed, 3 insertions(+), 3 deletions(-)
 
 diff --git a/vendor/libc/src/unix/mod.rs b/vendor/libc/src/unix/mod.rs
-index 238da24b..ffb05ee7 100644
+index ecee44411..7ae05f24e 100644
 --- a/vendor/libc/src/unix/mod.rs
 +++ b/vendor/libc/src/unix/mod.rs
-@@ -329,11 +329,11 @@ cfg_if! {
+@@ -337,11 +337,11 @@ cfg_if! {
          #[link(name = "c", cfg(not(target_feature = "crt-static")))]
          extern {}
      } else if #[cfg(target_env = "musl")] {
 +        #[link(name = "c")]
 +        extern {}
          #[cfg_attr(feature = "rustc-dep-of-std",
 -                   link(name = "c", kind = "static", modifiers = "-bundle",
-+                   link(name = "gcc", kind = "static-nobundle",
++                   link(name = "gcc", kind = "static", modifiers = "-bundle",
```

Comment:
`kind = "static-nobundle"` was removed. `static` defaults to the modifier `+bundle`, and setting the modifier `-bundle` behaves like ``static-nobundle` behaved before.